03_simon_says/simon_says.rb

```diff
@@ -1 +1,60 @@
-#write your code here
+def echo(greeting)
+    greeting.downcase
+end
+
+def shout(word)
+    word.upcase
+end
+
+def repeat(double_word, number_of_times = nil)
+    repeats = ""
+    if number_of_times != nil && number_of_times != 0
+        i = 0
+        while i < number_of_times - 1 do
+            repeats += double_word.downcase + " "
+            i += 1
+        end
+        repeats += double_word.downcase
+    end
+    if number_of_times == nil
+        repeats = double_word.downcase + " " + double_word.downcase
+    end
+    repeats = repeats.to_s
+end
+
+def start_of_word(word, number_of_first_letters)
+    word[0..number_of_first_letters - 1]
+end
+
+def first_word(string_of_words)
+    string_of_words.split.first
+end
+
+def titleize(untitleized_string)
```

Comment:
It is important to have good variable names to be easy understood, in this case we understand that we want to `titleize` something already. I suggest making the following change to simplify.
```suggestion
def titleize(string)
```